tests/unit/test__helpers.py

```diff
@@ -431,6 +440,17 @@ def test_w_struct(self):
 
         self.assertEqual(self._callFUT(value_pb, field_type), VALUES)
 
+    def test_w_numeric(self):
+        import decimal
+        from google.protobuf.struct_pb2 import Value
+        from google.cloud.spanner_v1.proto.type_pb2 import Type, NUMERIC
+
+        VALUE = decimal.Decimal("99999999999999999999999999999.999999999")
```

Comment:
Why name it with uppercases? Should it be `value`?

Reply:
It should, however I was matching with similar tests. I'll make a separate PR to fix this for all the tests.

Reply:
You can try to rename `from google.protobuf.struct_pb2 import Value` via: 

```
from relative_module import identifier as name
```

Or, change `VALUE` to `v` or `val`.